### DISCUSSION TITLE
Alarm on repeating log message

### DISCUSSION THREAD

**Discussion by martindeveloper** [apache/skywalking#9554] [General]
Hello,

Is there any way how to trigger alarm on a repeating log messages?
For example when service triggers 10x same exception (using regexp on exp. message for example) in 1 minute interval - send alarm email to support team.

Thanks.

**Comment by wu-sheng** [comment]
You could use `metrics` in `Extractor` of LAL(https://skywalking.apache.org/docs/main/latest/en/concepts-and-designs/lal/#extractor) to generate metrics. Then alerting could be triggered from the metrics.

**Reply by martindeveloper** [reply]
Sounds good, will try that. Thanks!
